src/components/scenegraph/Toolbar.js

```diff
@@ -89,7 +121,8 @@ export default class Toolbar extends React.Component {
     })
     .then(function (response) {
       let objects = response.data.objects;
-      self.setState({ objects });
+      let newObjectId = objects[objects.length-1].id+1;
```

Comment:
What is this part used for? And why are we saving it in the state? This won't always be accurately the new id of an object. If we have multiple scenes, then this could be like objects[objects.length-1].id + 13.